spt/utils/ent_utils.cpp

```diff
@@ -528,16 +528,7 @@ namespace utils
 #endif
 	}
 
-#if !defined(OE) && !defined(P2)
-	static int GetServerEntityCount()
-	{
-		auto engine_server = GetEngine();
-		if (!engine_server)
-			return 0;
-
-		return engine_server->GetEntityCount();
-	}
-
+#ifndef P2
 	static CBaseEntity* GetServerEntity(int index)
```

Comment:
Is it intentional that this function is now compiled on OE?

Reply:
It was intentional but OE is missing the necessary patterns for the tas_aim code anyway, so it hasn't been tested. I set it like that to futureproof it since there's nothing stopping it from compiling under OE, so I figure it would also work there.